RELEASE.md

```diff
@@ -20,6 +20,7 @@
 ## Documentation changes
 * Improved documentation for configuring dataset parameters in the data catalog
 * Extended documentation with an example of logging customisation at runtime
+* Added docstrings in three private functions in `kedro/utils.py`
```

Comment:
`docstrings` seem to be a valid plural: PEP [here](https://peps.python.org/pep-0257/).